client/checkout/blocks/index.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { registerPaymentMethod } from '@woocommerce/blocks-registry';
+
+/**
+ * Internal dependencies
+ */
+import { PAYMENT_METHOD_NAME } from '../constants.js';
+import { getConfig } from './../utils.js';
+import WCPayAPI from './../api';
+import WCPayFields from './fields.js';
+
+// Create an API object, which will be used throughout the checkout.
+const api = new WCPayAPI( {
+	publishableKey: getConfig( 'publishableKey' ),
+	accountId: getConfig( 'accountId' ),
+} );
+
+// Add the payment method to the blocks registry.
+registerPaymentMethod(
+	( PaymentMethodConfig ) => new PaymentMethodConfig( {
+		name: PAYMENT_METHOD_NAME,
+		content: <WCPayFields api={ api } />,
+		edit: <WCPayFields api={ api } />,
+		canMakePayment: () => true,
+		paymentMethodId: PAYMENT_METHOD_NAME,
+		label: __( 'Credit Card', 'woocommerce-payments' ),
```

Comment:
Payment methods can show an icon – might be nice to have Woo Pay icon on the tab. See example in [Stripe CC](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/main/assets/js/payment-method-extensions/payment-methods/stripe/credit-card/index.js#L41) or cheque. 

<img width="643" alt="Screen Shot 2020-07-10 at 4 45 22 PM" src="https://user-images.githubusercontent.com/4167300/87117479-c0ae8080-c2cc-11ea-84fc-da068cb9a82f.png">

Reply:
We're removing payment method icons, unless it's a consumer brand, such as paypal - so WC pay won't need a logo. 👍